app/views/shared/_cross_core_facility_abbreviation_order_details.html.haml

```diff
@@ -0,0 +1,8 @@
+- if order_detail.order.cross_core_project.present?
+  - facility_abbreviation = order_detail.order.cross_core_project.facility.abbreviation
+  - facility_abbreviation = facility_abbreviation.length > 6 ? facility_abbreviation.truncate(7) : facility_abbreviation
```

Comment:
Any reason not to just truncate everything?
```suggestion
  - facility_abbreviation = order_detail.order.cross_core_project.facility.abbreviation.truncate(7)
```

Reply:
`...` counts as part of the 6 chars. That means that if I have ABCDE, it displays "ABC..." instead of "ABCDE"

Reply:
You're right. I had tried a couple of options and saw something weird but now I couldn't reproduce it now, so I think I must have messed something up.